src/models/pats.rs

```diff
@@ -104,15 +104,15 @@ bitflags::bitflags! {
         const ORGANIZATION_DELETE = 1 << 38;
 
         const ALL = 0b111111111111111111111111111111111111111;
-        const NOT_RESTRICTED = 0b1111111100000011111111111111100111;
+        const NOT_RESTRICTED = 0b1111111110000000111111111111111111100111;
```

Comment:
Well this is kinda terrifying... glad it's tested now

Do we care that `NOT_RESTRICTED` has an extra 1? It specifies 40 bits where ALL specifies 39.

Reply:
Ill do a recount. I think they should be the same, but the test should be able to catch it so I'll look at that again

Reply:
It's apparent if you line them up in something monospaced like VSCode: 
![image](https://github.com/modrinth/labrinth/assets/4239887/ca67e461-2419-4b13-84e7-ab98e16be3e7)